queries/zathurarc/highlights.scm

```diff
@@ -0,0 +1,36 @@
+; General
+(path) @string.special.path
+
+(option) @variable.builtin
+
+(command) @keyword
+
+(include_directive
+  (command) @keyword.import)
+
+(mode_name) @type.builtin
+
+(key_name) @constant
```

Comment:
```suggestion
(key) @constant
```
`key_name` doesn't apply to regular keys.

Reply:
Done.